src/Lecture3.hs

```diff
@@ -79,7 +80,8 @@ Tuesday
   would work for **any** enumeration type in Haskell (e.g. 'Bool',
   'Ordering') and not just 'Weekday'?
 -}
-next = error "TODO"
+next :: Weekday -> Weekday
+next = succ
```

Comment:
That's almost correct! Unfortunately, `succ` on `Sunday` throws an exception 😞 
So you need to handle this case separately.